compiler_opt/rl/compilation_runner.py

```diff
@@ -152,8 +153,13 @@ def start_cancellable_process(
     TimeoutExpired: if the process times out.
     ProcessKilledError: if the process was killed via the cancellation token.
   """
+  command_env = os.environ.copy()
```

Comment:
can you make this configurable via an abseil flag? I'd just do "quiet" (default) vs not quiet - i.e. no need to elaborate on other values of TF_CPP_MIN_LOG_LEVEL or anything.

Reply:
Added a `--quiet`/`--noquiet` flag that toggles this.